config/settings.config.php

```diff
@@ -124,4 +122,82 @@
 	// Used in the admin settings to lock certain features
 	'featurelock' => [
 	],
+
+	// Mapping the log channels to the different log handlers
+	'log_channel' => [
+		'index' => ['default'],
+		'console' => ['default'],
+		'worker' => ['default'],
+		'daemon' => ['default'],
+		'db' => ['default'],
+		'auth_ejabbered' => ['ejabbered'],
+		'performance' => ['profiler'],
+		'develop' => ['develop'],
+	],
+
+	// Default log handling in Friendica
+	'log_handler' => [
+
+		// Standard Log handler in Friendica
+		'default' => [

```

Comment:
Hm.. I thought that. But I don't know a better way :-( .. Any guess?

Reply:
At the very least the console command has to be changed to prevent overwriting array values and to display them correctly.

Reply:
So you would prefer a alternative solution to the array? just to be clear.
=> Otherwise I'd start refactoring parts of the `Config` command for array usage.

Solutions I know:
1. Storing the `log_channel` and `log_handler` in different type of configs (Not per `Config`)
2. Refactoring the `Config` command
3. Denormalize the config dynamically:
```php
$config = [
	'log_handler' => [
		'names' => ['default', 'profile'],
		'default.type' => 'stream',
		'default.logfile' => 'friendica.log',
		[...]
		'profiler.type' => 'stream',
		'profiler.logfile' => 'profiler.log',
		[...]
	];
];
```

I think solution 3 is the one with the very least impact on the `Config`, but for me it sounds like breaking a little bit the normalization in arrays. I would write a new class `src\Object\HandlerConfig` which knows it's fields (name, type, desc, loglevel, logfile, ...) and will automatically loads it's values based on the pattern `Config::get('log_handler', '%name%.%field%')` and loaded because of `Config::get('log_handler', 'names')`. Sounds possible for me..

So I'd like to start with solution 3 if you support this way..